jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/service/ResourceApiService.java

```diff
@@ -173,24 +87,24 @@ public void deleteResource(UUID tenantId, UUID id) {
     }
   }
 
-  public T updateResource(UUID tenantId, UUID id, T payload) {
+  public T updateResource(T entity) {
 
     var transactionDefinition = new DefaultTransactionDefinition();
     var transactionStatus = this.transactionManager.getTransaction(transactionDefinition);
```

Comment:
Is it not the case if we were to annotate the enclosing method with @Transacrional, there wouldn't be a need for explicit transaction management (see my related comment in TenantRepositoryImpl class)

Reply:
This is part of the next subtask eahw-2545 to make the methods use the annotation instead of manually handling.